crowdin.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Your Crowdin credentials
+#
+"project_id_env": "CROWDIN_PROJECT_ID"
+"api_token_env": "CROWDIN_PERSONAL_TOKEN"
+"base_path": "."
+"base_url": "https://api.crowdin.com"
+
+#
+# Choose file structure in Crowdin
+# e.g. true or false
+#
+"preserve_hierarchy": true
```

Comment:
What would false look like in crowdin?
Would it still have the file names but all next to each other?
I think that would make it a bit easier for translating, if it's mapped back to the repo properly